ruby/red-arrow/lib/arrow/raw-tensor-converter.rb

```diff
@@ -41,12 +41,22 @@ def convert
       case @raw_tensor
       when Buffer
         @data = @raw_tensor
+      when String
+        @data = Arrow::Buffer.new(@raw_tensor)
       else
         @shape ||= guess_shape
         build_buffer
+        unless @strides.nil?
+          message = "strides: must not be specified: #{@strides.inspect}"
```

Comment:
How about `strides: is only accepted with a Buffer or String raw_tensor`?

Reply:
OK.